src/Raven.Server/Web/Operations/OperationsHandler.cs

```diff
@@ -29,6 +29,10 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
                     writer.WritePropertyName(nameof(GetNextOperationIdCommand.NodeTag));
                     writer.WriteString(Server.ServerStore.NodeTag);
                     writer.WriteEndObject();
+
```

Comment:
we have to the addition of Operations (`Database.Operations.AddOperation()`) as well.

Reply:
Done.
Notice: the  requestHandler propperty marked as [CanBeNull] because of Migrator.StartMigratingSingleDatabase() & BasicOperationsTests which are not inherit any handler